recipes/libxmlsec1/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+CONFIGURE="./configure --prefix=$PREFIX --disable-dependency-tracking --disable-crypto-dl --disable-apps-crypto-dl"
+$CONFIGURE
```

Comment:
Please, just execute that, or do you really need this variable in your env?
Just do
```
./configure --prefix=$PREFIX \
                  --disable-dependency-tracking \
                  --disable-crypto-dl \
                  --disable-apps-crypto-dl
```